editor/scene_tree_dock.cpp

```diff
@@ -391,6 +397,109 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 
 			tree->ensure_cursor_is_visible();
 
+		} break;
+		case TOOL_CUT:
+		case TOOL_COPY: {
+
+			const String copy_temp_file = EditorSettings::get_singleton()->get_cache_dir().plus_file("__node_copy.tscn");
+
+			if (current_option == TOOL_CUT && !profile_allow_editing)
+				break;
+
+			List<Node *> selection = editor_selection->get_selected_node_list();
+
+			if (selection.size() > 1) {
+				if (current_option == TOOL_CUT) {
+					accept->set_text(TTR("Can't cut  more than one node."));
```

Comment:
```suggestion
					accept->set_text(TTR("Can't cut more than one node."));
```

Also, it would be better to use `EditorNode::get_singleton()->show_warning(…)` to display error dialogs, which makes for shorter code.

Reply:
Fixed.

Looks like SceneTreeDock uses the `accept` dialog for warnings everywhere. This could be fixed one day.